Lps 88938

andresfulla opened the pull request:
@thektan Messed up pull https://github.com/andresfulla/liferay-portal/pull/8 😔

Last commit takes care of formatting issues https://github.com/andresfulla/liferay-portal/pull/8#discussion_r246615042

istvansajtos committed:
LPS-84703 journal-service: Do not index localized fields without using proper language analyzers; also, Search in all available locales for JournalArticles

https://issues.liferay.com/browse/LPS-84703

Co-authored-by: Adam Victor Nazareth Brandizzi <adam.brandizzi@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>
Co-authored-by: Eudaldo Alonso <eudaldo.alonso@liferay.com>
Co-authored-by: Istvan Sajtos <istvan.sajtos@liferay.com>
Co-authored-by: weicao <wade.cao@liferay.com>

istvansajtos committed:
LPS-86663 portal-search-test: Add test for LPS-85702

https://issues.liferay.com/browse/LPS-86663

lipusz committed:
LPS-86663 portal-search-test: Simplify (...)

1. Rename the new test cases to better refer to what they are actually testing (IPP queuing)
2. Pull down the logic into one common method as they are different in only 1 line
3. Use Dictionary for properties
4. Use serviceRegistration.unregister() just like in testNullIndexerIndexerPostProcessor

https://issues.liferay.com/browse/LPS-86663

arboliveira committed:
LPS-84703 portal-search-test: Test: Multi Language Search for title, content and description on Journal Articles and Document Library Files simultaneously; new localizaztion util.

https://issues.liferay.com/browse/LPS-84703

Co-authored-by: Adam Victor Nazareth Brandizzi <adam.brandizzi@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>
Co-authored-by: Eudaldo Alonso <eudaldo.alonso@liferay.com>
Co-authored-by: Istvan Sajtos <istvan.sajtos@liferay.com>
Co-authored-by: weicao <wade.cao@liferay.com>

arboliveira committed:
LPS-84703 journal-test-util: Test multi languag search.

- Deprecate classes moved to portal-search-test-util; add Description to JournalArticle blueprint
- Plain content field not indexed anymore, only translations
- Match any indexed translation regardless of current display language
- Search multiple sites with different language settings simultaneously
- No summary highlighting can be rendered when result is being displayed in a language that is not the same language of the localized snippets matched and returned by the search engine

https://issues.liferay.com/browse/LPS-84703

Co-authored-by: Adam Victor Nazareth Brandizzi <adam.brandizzi@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>
Co-authored-by: Eudaldo Alonso <eudaldo.alonso@liferay.com>
Co-authored-by: Istvan Sajtos <istvan.sajtos@liferay.com>
Co-authored-by: weicao <wade.cao@liferay.com>

arboliveira committed:
LPS-84703 journal-test: build.gradle

https://issues.liferay.com/browse/LPS-84703

Co-authored-by: Adam Victor Nazareth Brandizzi <adam.brandizzi@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>
Co-authored-by: Eudaldo Alonso <eudaldo.alonso@liferay.com>
Co-authored-by: Istvan Sajtos <istvan.sajtos@liferay.com>
Co-authored-by: weicao <wade.cao@liferay.com>

Commit:
LPS-86144 Implement clay cards in journal

ealonso committed:
LPS-86144 Apply changes

ealonso committed:
LPS-86144 SF

ealonso committed:
LPS-86144 Adds actions

ealonso committed:
LPS-86144 Uses clay dropdown-actions to show the actions on the different views

ealonso committed:
LPS-86144 Rename

ealonso committed:
LPS-86144 Apply changes

robertoDiaz committed:
LPS-88498 Add missing class

robertoDiaz committed:
LPS-88498 Revert "LPS-87497 Rename"

This reverts commit 96319b87b6ccc5c6dec27eb52ab783f5ecefde7a.

robertoDiaz committed:
LPS-88498 Revert "LPS-87497 Fix gradle"

This reverts commit 3249b5cd524c22f18dc5a36efe80ff50310f029e.

robertoDiaz committed:
LPS-88498 Revert "LPS-87497 VIEW entities with BackgroundTask classname is allowed"

This reverts commit 442c5ab78784ec3a43b4d430f48cb6a21278ed0f.

brandizzi committed:
LPS-84703 portal-search-api: Create a helper to get localized names.

https://issues.liferay.com/browse/LPS-84703

brandizzi committed:
LPS-84703 portal-search: Implement helper to get localized names.

https://issues.liferay.com/browse/LPS-84703

jorgediaz-lr committed:
LPS-88571 Avoid NPE in case group doesn't exists

Commit:
LPS-86364 Add missing language key

jorgediaz-lr committed:
LPS-88571 The DLFileEntryType with key = BASIC-DOCUMENT has companyId=0, groupId=0 and fileEntryTypeId=0 so in case group doesn't exists we should execute fetchExistingFileEntryType (old code before LPS-54392)

jorgediaz-lr committed:
LPS-88571 We should call group.getGroupId() instead of using groupId in all iterations

Commit:
LPS-86364 Re-word language key per LPS-80618 to use the terms "widget", "application", and "portlet" appropriately in UI messages

brandizzi committed:
LPS-84703 portal-search-elasticsearch6-impl: Do not expand  localized field names; instead, expect to receive all field names alrady processed.

https://issues.liferay.com/browse/LPS-84703

brandizzi committed:
LPS-84703 journal-test: Ensuring multi language search works.

https://issues.liferay.com/browse/LPS-84703

Commit:
LPS-88329 Getting started with multi contributor builder

Commit:
LPS-88735 Access ClassName value directly

joshchong committed:
LPS-84703 Remove obsolete functional test assertion

Commit:
LPS-88329 Extract configuration logic to Display Context class

Commit:
LPS-88329 Settling on two contributors editor

izaera committed:
LPS-83808 Hides module name logic from public API to minimize future version increases if implementation changes

Commit:
LPS-87597 Simplify and make executors internal

Commit:
LPS-87597 Update permissions

Commit:
LPS-87597 Add language term

Commit:
LPS-87597 buildLang

Commit:
LPS-87597 Add method to save model permissions for Data Definition

Commit:
LPS-87597 Update GraphQL schema

Commit:
LPS-87597 Add GraphQL data fetcher

marcelabc committed:
LPS-87597 Add method to save permissions for Data Definition

marcelabc committed:
LPS-87597 Update GraphQL schema

marcelabc committed:
LPS-87597 Add GraphQL data fetcher

Commit:
LPS-86144 Implement default event handler for cards in journal

jbalsas committed:
LPS-83808 Isolates script execution to avoid exceptions thrown inside an application to affect unrelated applications

jbalsas committed:
LPS-83808 Sets href to empty string as default, as undefined is not a valid value

Commit:
LPS-86144 Destroy components on navigation

Commit:
LPS-87597 Remove logging and handle exception

Commit:
LPS-87597 Add integration tests

Commit:
LPS-88791 Add rest bundle for change tracking with osgi configuration

Commit:
LPS-88791 Need to pass user and company ids separately

Commit:
LPS-88791 Create configuration request and response dtos

Commit:
LPS-88791 Create generic error reponse dto

Commit:
LPS-88791 Create application and configuration resource

Commit:
LPS-88791 Handle exceptions

Commit:
LPS-88894 Add content type and it's language key to the configuration

Commit:
LPS-88894 Update the tooling for content type

Commit:
LPS-88894 Update the journal configuration

Commit:
LPS-86144 Extract defaultEventHandler name to displayContext

izaera committed:
LPS-83808 Fix baseline increasing version number of taglib’s OSGi package

Commit:
LPS-88329 Updating CSS

Commit:
LPS-88329 stop propagation of change when not editing at CriteriaGroup level

jbalsas committed:
LPS-83808 Fix wrapperId generation and enhance internal state handling

shuyangzhou committed:
LPS-89139 Remove AggregatePredicateFilter

shuyangzhou committed:
LPS-89139 Remove PrefixPredicateFilter

shuyangzhou committed:
LPS-89139 Remove JavaScriptPortletResourcePredicateFilter

shuyangzhou committed:
LPS-89139 Remove DDMFormFieldValuePredicateFilter

shuyangzhou committed:
LPS-89139 Inline MapUtil.entryKeyPredicateFilter() and MapUtil.entryValuePredicateFilter()

shuyangzhou committed:
LPS-89139 Update MapUtil to use Predicate

shuyangzhou committed:
LPS-89139 Update ListUtil to use Predicate

shuyangzhou committed:
LPS-89139 Semantic versioning

Commit:
LPS-88894 Rename to JournalCTConfigurationRegistrar as it is registering the whole journal app

Commit:
LPS-88894 Make the resource service prototype scope

Commit:
LPS-88894 Reference registered configurations

Commit:
LPS-88329 Formatting issues

Commit:
LPS-88894 Update response DTO with new information

Commit:
LPS-88894 Update resource to return supported content types

Commit:
LPS-88329 Recovering modelLabel and removing query printing

izaera committed:
LPS-83808 Move soy renderers to their own module

ealonso committed:
LPS-86144 No needed

Commit:
LPS-88329 Passing propertyKey to nested CriteriaGroup

Commit:
LPS-88938 options as a pseudotype for the front

Commit:
LPS-87597 Replace internal OSGi components with simple java object

shuyangzhou committed:
LPS-89139 Update ArrayUtil to use Predicate

shuyangzhou committed:
LPS-89139 Regenerate IW

shuyangzhou committed:
LPS-89139 Update ComboServletStaticURLGenerator to use Predicate

shuyangzhou committed:
LPS-89139 Remove PredicateFilter

lanceji92674957 committed:
LPS-88977 Replace mock with wrapper

ealonso committed:
LPS-86144 Adds statuses

izaera committed:
LPS-85962 Propagate commit 466de15c043ef3e0c99d6ddda087bb6b3936f1aa as it was lost with rebase

Commit:
LPS-88791 Avoid using dto, using model instead

izaera committed:
LPS-83808 SF

Commit:
LPS-88329 classname module, small readability issues and better formatting

Commit:
LPS-88938 Better formatting

izaera committed:
LPS-83808 Move soy renderer API to its own module to comply with SF

balazssk committed:
LPS-88951 Adding p_l_id to URLs so that themeDisplays can be initialized correctly

Commit:
LPS-88938 removing unnecessary wrapper and adding some basic tests

GergoMathe committed:
LPS-88848 Create JournalArticleCTServiceWrapper with add and update operations

jcampoy committed:
LPS-88931 Fix parameter name. See UpdateDDMStructureMVCActionCommand or AddDDMStructureMVCActionCommand.

Commit:
LPS-88894 Fix rebase issues and SF

marcelabc committed:
LPS-87597 Handle a new exception

Commit:
LPS-88938 using classnames

thektan commented:
Is this still a wip? Let me know when it's good to do a full review. 😄

Commit:
LPS-88938 Removing unnecessary css class

andresfulla commented:
> Is this still a wip? Let me know when it's good to do a full review. 😄

You can go ahead and full review it. Also, I'm opening another one on top of this one for some primitive types.

pavel-savinov committed:
LPS-89004 Add empty widgets panel to the fragments editor sidebar

ealonso committed:
LPS-86144 Adds necessary css styles

IstvanD committed:
LPS-88995 DDM upgrade to 7.1 fails with NPE due to DDMContent table's stats being locked

jcampoy committed:
LPS-88931 Do not use the BeanParamUtil because the bean property name is not equals to the parameter name.

thektan commented on modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/TypedInput.es.js:
```diff
@@ -0,0 +1,59 @@
+				)}
+				selected={value}
+			/>;
+	}
```
Let's add new lines between each method.

thektan commented on modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/TypedInput.es.js:
```diff
@@ -0,0 +1,59 @@
+		if (onChange) {
+			onChange(value);
+		}
+	}
```
Let's use the arrow function here so we don't have to call `.bind(this)`.

thektan commented on modules/apps/segments/segments-web/test/js/components/criteria_builder/TypedInput.es.js:
```diff
@@ -0,0 +1,60 @@
+import TypedInput from 'components/criteria_builder/TypedInput.es';
+import {cleanup, render} from 'react-testing-library';
+
+const connectDnd = jest.fn(el => el);
```
I think we can remove this.

thektan commented on modules/apps/segments/segments-web/test/js/components/criteria_builder/TypedInput.es.js:
```diff
@@ -0,0 +1,60 @@
+}, {
+	label: 'Some Other Option',
+	value: 'someOtherOption'
+}];
```
Csf isn't checking the test files, but let's still format this as:
```javascript
const options = [
    {
	label: 'Default Value',
	value: 'defaultValue'
    },
    {
	label: 'Some Other Option',
	value: 'someOtherOption'
    }
];
```

Preston-Crary committed:
LPS-89024 Update tasks portlet to use current taglibs.

Preston-Crary committed:
LPS-89024 Remove util-taglib-compat.

Preston-Crary committed:
LPS-89024 Remove bnd references to util-taglib-compat.

thektan reviewed (changes requested):
Looks like just formatting changes left. 👍

Preston-Crary committed:
LPS-89024 Versioning.

Commit:
LPS-88329 removing unnecessary getCN & adding default prop

Preston-Crary committed:
LPS-89024 Add back taglibs still in use.

thektan committed:
LPS-88329 Use arrow function for `this` context

thektan committed:
LPS-88329 Keep component name consistent with file name

thektan committed:
LPS-88329 Format for code readability

Add new lines to prevent variable sorting and rename some variables to be more verbose

thektan committed:
LPS-88329 Use conjunction constant

thektan committed:
LPS-88329 Remove unused import

Preston-Crary committed:
LPS-89024 SF.

thektan committed:
LPS-88329 Unnecessary setting state with no change

From comment: https://github.com/bryceosterhaus/liferay-portal/pull/13#discussion_r246483807

thektan committed:
LPS-88329 setState will only update the specified state

Renaming prevProps to props since setState doesn't change props

From comment: https://github.com/bryceosterhaus/liferay-portal/pull/13#discussion_r246486576

thektan committed:
LPS-88329 Reduce unnecessary props, returns, and braces

shuyangzhou committed:
LPS-89139 Update com.liferay.portal.kernel.portlet package version locking range

Preston-Crary committed:
LPS-89024 Remove util-taglib-compat from gitignore.

thektan commented:
Also let's add a commit message description to https://github.com/andresfulla/liferay-portal/pull/9/commits/f2f8c917bf291fad0e839563e0f88ddacce224cc

tinatian committed:
LPS-89033 Add private method, no logic change

tinatian committed:
LPS-89033 Auto SF

tinatian committed:
LPS-89003 Add support to queue messages if current message listener is running. At the same time, only one thread is processing the message.

ealonso committed:
LPS-86144 Include modified date into the subtitle, like in old cards

ealonso committed:
LPS-86144 Replace primary for info, because primary is not a valid value for labels

izaera committed:
LPS-83808 Fix versions

pavel-savinov committed:
LPS-89004 Populate editor context with existing categories and widgets

ling-alan-huang committed:
LPS-84119 Add unary operators(-/+)

Commit:
LPS-88329 Renaming snapshots

Commit:
LPS-88329 setState improvements

Commit:
LPS-88791 refactor the whole bundle to fit patterns

pavel-savinov committed:
LPS-89004 Show available widgets in the corresponding panel

Commit:
LPS-88791 Pattern - use string literals instead of constants even thought it's more prone to errors

pavel-savinov committed:
LPS-89004 Apply styles to the widgets panel as defined in the UX mockups

marcelabc committed:
LPS-87597 Fix up rebase changes

pavel-savinov committed:
LPS-89004 Language key

pavel-savinov committed:
LPS-89004 Buildlang

Commit:
LPS-88329 root conjunction change

Commit:
LPS-88329 right formatting

Commit:
LPS-88938 better formatting & class field methods

4lejandrito committed:
LPS-89056 Adding edit categories bulk selection action

ericyanLr committed:
LPS-87946 update jodconverter dependency

shuyangzhou committed:
LPS-89139 Bump SB portal kernel dependency to default.

shuyangzhou committed:
LPS-89139 Revert LPS-77110 1fe38a052f2e9e5f4365e158d6979a59d01f0b70, this broke SB generated code's FQDN simplification.

thektan committed:
LPS-88329 Add new lines between statements

marcelabc committed:
LPS-87597 Refactor integration tests after rebase

thektan committed:
LPS-88329 Rename editing state to editingId

The name  `editing` suggests a boolean where in other components `editing` is already used as a boolean. Since this value is an identifier to which contributor is being edited, `editingId` is more appropriate.

Preston-Crary committed:
LPS-89024 Fix compile errors from removing util-taglib-compat.

tinatian committed:
LPS-89033 Add unit test to expose bug

Preston-Crary committed:
LPS-89024 Remove unused jsps.

thektan committed:
LPS-88329 Add eslint rule to check for handler method names

thektan committed:
LPS-88329 Fix event handler method names

thektan committed:
LPS-88329 Add default prop for initialMembersCount

Commit:
LPS-88791 Rename application

brianchandotcom committed:
LPS-83808 Hugo SF, at least for new code. I suppose you could make ant format-source-current-branch only check certain things like this, since it can detect new files. This is low low priority.

brianchandotcom committed:
LPS-83808 Wordsmith

ling-alan-huang committed:
LPS-84119 Add testcase

brianchandotcom committed:
LPS-88791 SF

brianchandotcom committed:
LPS-88791 Wordsmith

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as 7b1a93445ebd0b0e20288d1f1f8b4985f7feb9fa

hhuijser committed:
LPS-84119 Simplify

hhuijser committed:
LPS-84119 SF

achaparro committed:
LPS-89064 No longer needed after LPS-88901

ealonso committed:
LPS-89064 Auto SF

ling-alan-huang committed:
LPS-84119 Improve logic

ling-alan-huang committed:
LPS-84119 Add testcase

jkappler committed:
LPS-88978 There's no need to check for required primary site navigation menu since we have the pages hierarchy as the default form of navigation

jkappler committed:
LPS-88978 Treat primary navigation as any other type

adolfopa committed:
LPS-89025 Parameter order was inverted

jkappler committed:
LPS-88978 No longer needed, since the exception isn't thrown anymore

Commit:
LPS-89057 - Regen package-lock files

jkappler committed:
LPS-88978 New test to check that the previous navigation menu with a specific type reverts to default after adding a new one for the type

jkappler committed:
LPS-88978 The tests are no longer needed, since no exception is thrown when deleting a primary navigation menu

ealonso committed:
LPS-89004 SF

jkappler committed:
LPS-88978 We don't want to mark as primary when creating a new site navigation menu

ealonso committed:
LPS-89004 Avoid unnecessary loop if title is not null

ealonso committed:
LPS-89004 Uses servlet context like in /view_categories.jsp

GergoMathe committed:
LPS-88848 Throw CTException when registering a model change fails

ealonso committed:
LPS-89004 SF

GergoMathe committed:
LPS-88848 Handle exception in JournalArticleCTServiceWrapper, don't fail for duplicate addition

GergoMathe committed:
LPS-88848 Fix exception name

GergoMathe committed:
LPS-88848 Autogenerated

GergoMathe committed:
LPS-88848 Apply renamed exception

GergoMathe committed:
LPS-88848 Move JournalArticleCTServiceWrapper to journal-change-tracking module

4lejandrito committed:
LPS-89056 Adding REST endpoint

4lejandrito committed:
LPS-89056 Improving names

4lejandrito committed:
LPS-89056 Using a single method

Commit:
LPS-88848 SF

Commit:
LRQA-46011 Update export import portlet test case

Commit:
LPS-88848 Rename to *LocalServiceWrapper because it's wrapping the local service

adolfopa committed:
LPS-82785 Clean up images on calls to updateAndCheckIn

adolfopa committed:
LPS-82785 Update dependencies

pavel-savinov committed:
LPS-88978 Remove unused exception

pavel-savinov committed:
LPS-88978 Semantic versioning

Commit:
LRQA-46011 Fix script error

Commit:
LPS-88848 Fix compile error on test

Commit:
LRQA-45807 Enable acceptance test case

Commit:
LPS-88323 Use ecma5

brianchandotcom committed:
LPS-88951 p_l_id is one of those special vars

brianchandotcom committed:
LPS-82785 Rename

brianchandotcom committed:
LPS-88674 master now 2.0.0

brianchandotcom committed:
LPS-88848 Wordsmith

brianchandotcom committed:
LPS-88848 It is not a CTLocalService, it is a JournalArticleLocalService, so let's call it CTJournalArticleLocalService. This makes it consistent with modules/apps/mentions/mentions-service/src/main/java/com/liferay/mentions/internal/service/MentionsBlogsEntryServiceWrapper.java

brianchandotcom committed:
LPS-89004 as used

brianchandotcom committed:
LPS-89004 Rename

brianchandotcom committed:
LPS-89004 Rename

brianchandotcom committed:
LPS-89004 Rename for consistency please

Preston-Crary committed:
LPS-89024 Update projects.

brianchandotcom committed:
LPS-87597 Rename

brianchandotcom committed:
LPS-87597 This is my 3rd time renaming this var (from 3 different pulls). Please proactively fix next time. Thx.

brianchandotcom committed:
LPS-87597 not needed

brianchandotcom committed:
LPS-87597 use else if here because it would be very weird to have both of those properties. If that happens, always take the first one vs. last one.

brianchandotcom committed:
LPS-87597 Add log, or else I get this error in SF: Unprocessed exception 'NoSuchRoleException', see LPS-36174: ./modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/executor/DEDataDefinitionSavePermissionsRequestExecutor.java 71 (Checkstyle:UnprocessedExceptionCheck)

petershin committed:
LPS-89126 Update outputs for the npmInstall task

petershin committed:
LPS-89126 Use latest gradle-plugins-node

marcelabc committed:
LPS-87597 Remove ELSE/IF because all these properties can be set in the same request

petershin committed:
LPS-89126 Use latest gradle-plugins-jsdoc

petershin committed:
LPS-89126 Use latest gradle-plugins-js-transpiler

petershin committed:
LPS-89126 Use latest gradle-plugins-js-module-config-generator

petershin committed:
LPS-89126 Use latest gradle-plugins-gulp

thektan commented on modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_sidebar/CriteriaSidebar.es.js:
```diff
@@ -57,9 +57,10 @@ class CriteriaSidebar extends Component {
+							({label, name, options, type}, index) => (
 								<CriteriaSidebarItem
 									className={`color--${propertyKey}`}
+									defaultValue={options && options.length ? options[0].value : ''}
```
Instead of having a `defaultValue` being passed into the `CriteriaSidebarItem` (who's only purpose is to pass along the `defaultValue` data), do you think it makes sense to set the default value within `_handleCriterionAdd` in `CriteriaGroup`?  Especially since we're always using the first option value as the default value, I don't think `CriteriaSidebarItem` needs to know about the default value.

brianchandotcom committed:
LPS-84703 as used

brianchiu committed:
LPS-86144 Update card ellipsis selector

joshchong committed:
LRQA-46014 Add reindex as a workaround due to LPS-89026

joshchong committed:
LRQA-46017 Enable gender field

joshchong committed:
LRQA-45997 Add UserLocalServiceTest to search test suites

brianchiu committed:
LPS-80199 Fix for tests with child pages

brianchiu committed:
LPS-80199 Rename

brianchiu committed:
LPS-80199 Have these click the tab rather than caret icon

marcelabc committed:
LPS-87597 Remove ELSE/IF because all these properties can be set in the same request

brianchiu committed:
Revert "LPS-88323 Use ecma5"

This reverts commit 1d4389ecdace24aa8d2a42a0a24be4fb313fe70b.

petershin committed:
LPS-84119 Update changelog

petershin committed:
LPS-86806 Update changelog

Commit:
LPS-87511 Open sidebar when rendering an empty Form Builder

Commit:
LPS-87511 Update unit tests

ruben-pulido committed:
LPS-88715 Fix path to file

ruben-pulido committed:
LPS-88715 Consistency: Use test name as title of document created

Commit:
LPS-88323 Use ecma5

thektan commented on modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/TypedInput.es.js:
```diff
@@ -0,0 +1,56 @@
+import getCN from 'classnames';
+
+class TypedInput extends React.Component {
+	_onChange = event => {
```
Let's also rename this to `_handleChange`.

brianchandotcom committed:
LPS-89126 prep next

brianchandotcom committed:
LPS-89139 SF

brianchandotcom committed:
LPS-89139 SF

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as dc61c09e96ec6cba24254ad117d54a0a4ef93fac

brianchiu committed:
LPS-80199 Simply because it is too complicated to fit into the mold

joshchong committed:
LRQA-46014 Add reindex to more flaky acceptance tests as workaround for LPS-89026

joshchong committed:
LRQA-46026 Reduce flakiness

hhuijser committed:
LPS-84119 SF, no need to use else if statement

hhuijser committed:
LPS-84119 Improve performance a tiny bit by not calling getChildCount

Commit:
LPS-88938 Better tests & formatting
